### ISSUE TITLE
VoiceOver - incorrect focus after dismissing keyboard

### ISSUE THREAD

**Issue by accessiblewebuk** [alphagov/accessible-autocomplete#371]
After typing in a selection, when the keyboard is dismissed, focus seems to jump to the 5th item in the list of results or to the heading. Would expect to jump to the first item in the list, otherwise users may not be aware of the preceding results and miss the one they want.

**Comment by vanitabarrett** [comment]
First step: confirm this is still an issue and that we can reproduce it.
